<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(224,255,255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="red_slider">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 0), stop:1 rgba(255, 0, 0, 1));</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="green_slider">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 255, 0, 0), stop:1 rgba(0, 255, 0, 1));</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="blue_slider">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 255, 0), stop:1 rgba(0, 0, 255, 1));</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="current_color">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color:#000000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="red">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;
font-weight:800;</string>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="green">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green;
font-weight:800;</string>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="blue">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:blue;
font-weight:800;</string>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="red_box">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="green_box">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="blue_box">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>481</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="rgb_pallet">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Franklin Gothic Medium&quot;;</string>
       </property>
       <property name="text">
        <string>rgb(0,0,0)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hex_pallet">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Franklin Gothic Medium&quot;;</string>
       </property>
       <property name="text">
        <string>#000000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
